Packs/FeedFireEye/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### FireEye Feed
+- Maintenance and stability enhancements.
+- Updated docker image to the latest version.
```

Comment:
```suggestion
- Upgraded the Docker image to demisto/python3:3.8.6.12176.
```

You can use the demisto-sdk update-release-notes and this format will be autogenerated for you.